pkg/integration/utils.go

```diff
@@ -27,6 +27,16 @@ func GetExitCode(err error) (int, error) {
 			return procExit.ExitStatus(), nil
 		}
 	}
+	// error if contained command not found/does not exist
```

Comment:
Can you explain this code here? If we have to check the output error message to "calculate" the exit code, then isn't something wrong since I thought the point of this PR was to make sure our exit codes were set properly?